cms/djangoapps/contentstore/tests/test_course_create_rerun.py

```diff
@@ -149,12 +158,13 @@ def test_course_creation_with_org_not_in_system(self, store):
                 'run': '2015_T2'
             })
             self.assertEqual(response.status_code, 400)
+            with self.assertRaises(InvalidOrganizationException):
+                get_organization_by_short_name("orgX")
             data = parse_json(response)
             self.assertIn(u'Organization you selected does not exist in the system', data['error'])
 
-    @patch.dict('django.conf.settings.FEATURES', {'ORGANIZATIONS_APP': True})
-    @ddt.data(ModuleStoreEnum.Type.mongo, ModuleStoreEnum.Type.split)
-    def test_course_creation_with_org_in_system(self, store):
+    @ddt.data(False, True)
+    def test_course_creation_for_known_organization(self, enable_strict_organizations):
```

Comment:
Which store we use really doesn't matter for the organizations-related test functions, so I've removed the mongo/split ddt cases for this test function, although I left it on the other test functions for the sake of minimizing the diff.